modules/broker-coordination/src/main/java/org/wso2/broker/coordination/rdbms/CoordinationConfiguration.java

```diff
@@ -43,15 +43,16 @@ public void setRdbmsCoordinationConfig(RdbmsCoordinationConfiguration rdbmsCoord
      */
     public static class RdbmsCoordinationConfiguration {
 
-        private String nodeId;
+        private String nodeId = "DEFAULT_ID";
 
-        private int heartbeatInterval;
+        private int heartbeatInterval = 5000;
 
-        private int coordinatorEntryCreationWaitTime;
-
-        private int eventPollingInterval;
+        private int coordinatorEntryCreationWaitTime = 3000;
 
         public String getNodeId() {
+            if (nodeId.equals("DEFAULT_ID")) {
```

Comment:
This will return different values for different invocations of the `getNodeId()'. We should ideally do the ramdom UUID generation in the place where we access the value(maybe in RdbmsCoordinationStrategy )?

Reply:
For the default value, I would prefer a single word like 'GENERATED'. WDYT?